src/viking.js

```diff
@@ -1,11 +1,72 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health=health
+        this.strength=strength
+    }
+    attack(){
+        return this.strength
+    }
+    receiveDamage(damage){
+        this.health-= damage
+
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name,health, strength){
+        super(health, strength)
+        this.name=name
+    }
+    receiveDamage(damage){
+        this.health-= damage
+        if(this.health>0){
```

Comment:
I would use a little bit more spacing here, just to make the code easier to read
```suggestion
        if (this.health>0) {
```